### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by sam-dahlia** [tailwindlabs/headlessui#2850] [General]
Hi folks,

I've been loving the headless library and using it in conjunction with tailwind CSS and tailwind components.
I unfortunately need a fix for a bug that's not included in the latest release. The [fix](https://github.com/tailwindlabs/headlessui/pull/2697) was committed just a few days after the last public version.

Do we know when the next public version will be released? I'd rather not deploy an `insider` dependency to prod.

Thank you!

**Comment by sam-dahlia** [comment]
Hi, 
Can I get some comments on this?
I'm starting to think this project may be abandoned at this point. 
Should I look at alternatives?

**Reply by RobinMalfait** [reply]
Hi!

Sorry for the late response. We are working on the next version as we speak with a bunch of new features, improvements and new components. This will also be used in our Catalyst UI kit: https://twitter.com/adamwathan/status/1729230772383473982

We are also updating the docs for the new features and components but we will have a release out hopefully very soon.

In the meantime, you can still use insiders 💪

Hope this brings some clarity!

**Reply by sam-dahlia** [reply]
Thanks for the response. Looking forward to the new update!
